src/hipoc/hipoc_program.cpp

```diff
@@ -44,7 +44,7 @@
 
 #include <unistd.h>
 
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_OPENCL_ENFORCE_COV3)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_OPENCL_ENFORCE_COV2)
```

Comment:
This change also requires updating the documentation.

Reply:
Done